tests/api/asserts.rs

```diff
@@ -37,9 +37,20 @@ pub async fn assert_auth_key_utf8(response: Response) -> AuthKey {
 // OK response
 
 pub async fn assert_ok(response: Response) {
-    assert_eq!(response.status(), 200);
-    assert_eq!(response.headers().get("content-type").unwrap(), "application/json");
-    assert_eq!(response.text().await.unwrap(), "{\"status\":\"ok\"}");
+    let response_status = response.status().clone();
+    let response_headers = response.headers().get("content-type").cloned().unwrap();
+    let response_text = response.text().await.unwrap();
+
+    let details = format!(
+        r#"
+   status: ´{response_status}´
+  headers: ´{response_headers:?}´
+     text: ´"{response_text}"´"#
+    );
```

Comment:
hey @da2ce7 , this is an interesting way to have a kind of "object literal" or "anonymous struct". It's related to https://github.com/torrust/torrust-tracker/pull/172

I would not include it in this PR unless we change all the asserts to follow the same pattern. Maybe the final solution could be a mix of this solution and what I proposed on that [PR](https://github.com/torrust/torrust-tracker/pull/172). In this solution, I still miss the error pointing to the origin line when the condition was not fulfilled.